arch/sim/src/sim/up_setjmp32.S

```diff
@@ -47,9 +47,9 @@
 # define SYMBOL(s) _##s
 #else
 #ifdef __ELF__
-# define SYMBOL(s) _##s
-#else
 # define SYMBOL(s) s
+#else
+# define SYMBOL(s) _##s
 #endif
 #endif
 
```

Comment:
How about:
#ifdef __ELF__
# define SYMBOL(s) s
#else
# define SYMBOL(s) _##s
#endif
Like you change:
#ifdef __ELF__
	.type	SYMBOL(up_setjmp), @function
#endif
Also for up_setjmp64.S

Reply:
do you mean to merge __CYGWIN__ block?
__CYGWIN__ seems to use _ prefix for 32-bit but not for 64-bit.
so I feel it's better to keep it separate.

Reply:
Yes, the separation is better.